### PR TITLE
Feature/Add CanCreateRelationship use case

### PR DESCRIPTION
**Pull request by britsta**
Add a description for the CanCreateRelationship use case and mention the use case in the affected "Establish Relationships" scenario.

### CODE REVIEW

File: _docs_integrate/establish-relationships.md [hunk 1]
```diff
@@ -112,6 +112,13 @@ If an App user wants to initiate a Relationship with the templator, the App user
 
 After the templator has created a RelationshipTemplate and made it available to another Identity, this Identity can use it to initiate a Relationship with the templator. For this reason, this other Identity is also referred to below as the initiator.
 
+### Check the Initiability of a Relationship
```
Comment (jkoenig134) [inline _docs_integrate/establish-relationships.md:115]: wtb "Check if the Relationship can be established"?

File: _docs_use-cases/use-case-transport-check-if-relationship-can-be-created.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+api_route_regex: ^PUT /api/v2/Relationships/CanCreate$
+# End automatic generation
+---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use case checks whether a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) can be created based on a received [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) without actually creating it.
+It makes sense to promptly execute this use case in order to prevent an App user from providing the data required to [initiate a Relationship]({% link _docs_integrate/establish-relationships.md %}#initiate-it-as-an-app-user) when this is currently not technically possible anyway.
```
Comment (Milena-Czierlinski) [inline _docs_use-cases/use-case-transport-check-if-relationship-can-be-created.md:46]: I think the focus target group here are integrators. The current wording then can be misleading, as they might think they must do something to prevent an App user from unnecessarily entering their data in a RelationshipTemplate. Therefore, I'd rephrase the motivation to fit the view of an integrator.

File: _docs_use-cases/use-case-transport-check-if-relationship-can-be-created.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+The Relationship would be established to the RelationshipTemplate's creator.
```
Comment (Milena-Czierlinski) [inline _docs_use-cases/use-case-transport-check-if-relationship-can-be-created.md:47]: I don't understand why this sentence is in this place.

File: _docs_use-cases/use-case-transport-check-if-relationship-can-be-created.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  -- The `templateId` does not resolve to a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) or the associated RelationshipTemplate was not cached correctly.<br>
+  -- A [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) with `"Pending"`, `"Active"`, `"Terminated"` or `"DeletionProposed"` as `status` already exists to the creator of the RelationshipTemplate.<br>
+  -- The [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) has already expired, which means that the timestamp specified in its `expiresAt` property has been exceeded.<br>
+  -- The potential initiator has already [decomposed the former Relationship]({% link _docs_integrate/terminate-relationships.md %}#decompose-a-relationship) to the creator of the RelationshipTemplate, but the RelationshipTemplate's creator has not yet decomposed it.<br>
```
Comment (Milena-Czierlinski) [inline _docs_use-cases/use-case-transport-check-if-relationship-can-be-created.md:64]: This is the case if a Relationship with `"DeletionProposed"` as `status` already exists to the creator of the RelationshipTemplate, which you mentioned in the second point. I do think it is good to point this out explicitly again, since I don't expect the average integrator to know this detail immediately. However, I'd either move this point directly below it or even combine it with the other point by adding another sentence with more details how a Relationship with `"DeletionProposed"` as `status` can exist (without being visible).

File: _docs_use-cases/use-case-transport-create-relationship-with-relationshiptemplate.md [hunk 5]
```diff
@@ -42,12 +42,17 @@ api_route_regex: ^POST /api/v2/Relationships$
...
+- `templateId` references the RelationshipTemplate that was received from its creator.
+- `creationContent` is usually an [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent) when the RelationshipTemplate's `content` is an ArbitraryRelationshipTemplateContent and the use case is applied manually and a [RelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) when the RelationshipTemplate's `content` is a RelationshipTemplateContent and the [Request Module]({% link _docs_explore/61-runtime.md %}#request-module) automatically takes care of the execution of the use case. It will be stored inside the Relationship and can be analyzed by the creator of the RelationshipTemplate to decide whether to [accept the Relationship]({% link _docs_use-cases/use-case-transport-accept-relationship.md %}).
```
Comment (Milena-Czierlinski) [inline _docs_use-cases/use-case-transport-create-relationship-with-relationshiptemplate.md:55]: Please split up this sentence, I am having trouble understanding it with all the "and"s. Also, use "if"s instead of "when"s.

File: _docs_integrate/establish-relationships.md (hunk 1 above)
Comment (britsta) [reply _docs_integrate/establish-relationships.md:115]: What's wrong with the current wording? ;) I like it better as it is now because it is shorter and contains the word "initiate" (which I feel is more appropriate wording in this "Initiate a Relationship" section of the guide).

Comment (jkoenig134) [reply _docs_integrate/establish-relationships.md:115]: I don't like the word `Initiability` that is never used again..

Comment (jkoenig134) [reply _docs_integrate/establish-relationships.md:115]: > is more appropriate wording in this "Initiate a Relationship"

then rename this to establish

Comment (Milena-Czierlinski) [reply _docs_integrate/establish-relationships.md:115]: I have never heard "initiability" before. Linguee doesn't know it either and it is also underlined in red here.

Comment (jkoenig134) [reply _docs_integrate/establish-relationships.md:115]: > I have never heard "initiability" before. Linguee doesn't know it either and it is also underlined in red here.

yeah, and this :D

Comment (britsta) [reply _docs_integrate/establish-relationships.md:115]: Good bye "Initiability", we (or I 🤣) will miss you. 💔

File: _docs_integrate/establish-relationships.md [hunk 6]
```diff
@@ -112,7 +112,7 @@ If an App user wants to initiate a Relationship with the templator, the App user
 
 After the templator has created a RelationshipTemplate and made it available to another Identity, this Identity can use it to initiate a Relationship with the templator. For this reason, this other Identity is also referred to below as the initiator.
 
-### Check the Initiability of a Relationship
+### Check the Possibility of a Relationship Initiation
```
Comment (Milena-Czierlinski) [inline _docs_integrate/establish-relationships.md:115]: If you want to keep the grammatical structure, use "feasibility" instead of "possibility". However, I still find this wording harder to understand than what @jkoenig134 suggested. If you don't like "establish" and want to keep "initiate", how about "Check if the Relationship can be initiated", instead?

Comment (britsta) [reply _docs_integrate/establish-relationships.md:115]: Of course I can change "possibility" to "feasibility". 👍 Regarding the other suggestions, I don't like it if we use sentences as titles and I am more a fan of noun sequences in titles. But if it is important to you, I can also change it.

Comment (Milena-Czierlinski) [reply _docs_integrate/establish-relationships.md]: Well, I think then it is your personal preference against mine. So all good, no deal breaker to me.

**Review by Milena-Czierlinski — APPROVED** [review]
